### DISCUSSION TITLE
Counter on the Collection layout as configurable

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#11863] [Feature Requests]
Can we prioritize Counter on the Collection page? It is very helpful to get quick insights

![unknown (3)](https://user-images.githubusercontent.com/36895131/155761626-4b5f0a0b-09e0-4890-8e39-200cd2eb7338.png)

**Comment by benhaynes** [comment]
Not sure what you mean by "prioritize"... but [it can be sponsored](https://docs.directus.io/getting-started/support/#sponsored-work), yes. 🙂

**Reply by Jeyakumarrathnaselvi** [reply]
like to fund this activity. not sure any limit on the amount

**Comment by w0kyj** [comment]
Additional Requirements from Live Chat:
1. expecting a configurable counter in each collection section itself. inside /settings/data-model since counters will be collection specific

2. Regarding calculation, It is similar to Metric feature in Insights

**Reply by Jeyakumarrathnaselvi** [reply]
- you want this configured from admin settings -- Is there a need/want to allow the end-user to change the panel?
end user need not change it

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
